src/api/python/cvc5.pxi

```diff
@@ -2395,245 +2620,259 @@ cdef class Solver:
             assert isinstance(args[0], Grammar)
-            result.cterm = self.csolver.getInterpolant(conj.cterm, (<Grammar ?> args[0]).cgrammar)
+            result.cterm = self.csolver.getInterpolant(
+                    conj.cterm, (<Grammar ?> args[0]).cgrammar)
         return result
 
 
     def getInterpolantNext(self):
         """
-        Get the next interpolant. Can only be called immediately after
-        a succesful call to get-interpolant or get-interpolant-next. 
-        Is guaranteed to produce a syntactically different interpolant wrt the
-        last returned interpolant if successful.
+            Get the next interpolant. Can only be called immediately after
+            a succesful call to get-interpolant or get-interpolant-next.
+            Is guaranteed to produce a syntactically different interpolant wrt
+            the last returned interpolant if successful.
 
-        SMT-LIB:
+            SMT-LIB:
 
-        .. code-block:: smtlib
+            .. code-block:: smtlib
 
-            ( get-interpolant-next )
+                ( get-interpolant-next )
 
-        Requires to enable incremental mode, and 
-        option :ref:`produce-interpolants <lbl-option-produce-interpolants>` to be set to a mode different from `none`.
+            Requires to enable incremental mode, and option
+            :ref:`produce-interpolants <lbl-option-produce-interpolants>` to be
+            set to a mode different from `none`.
 
-        .. warning:: This method is experimental and may change in future
-                     versions.
+            .. warning:: This method is experimental and may change in future
+                         versions.
 
-        :param output: the term where the result will be stored
-        :return: True iff an interpolant was found
+            :param output: The term where the result will be stored.
+            :return: True iff an interpolant was found.
         """
         cdef Term result = Term(self)
         result.cterm = self.csolver.getInterpolantNext()
         return result
-        
+
     def getAbduct(self, Term conj, *args):
-        """Get an abduct.
+        """
+            Get an abduct.
 
-        SMT-LIB:
+            SMT-LIB:
 
-        .. code-block:: smtlib
+            .. code-block:: smtlib
 
-            ( get-abduct <conj> )
-            ( get-abduct <conj> <grammar> )
+                ( get-abduct <conj> )
+                ( get-abduct <conj> <grammar> )
 
-        Requires to enable option :ref:`produce-abducts <lbl-option-produce-abducts>`.
+            Requires to enable option :ref:`produce-abducts
+            <lbl-option-produce-abducts>`.
 
-        Supports the following variants:
+            Supports the following variants:
 
-        - ``Term getAbduct(Term conj)``
-        - ``Term getAbduct(Term conj, Grammar grammar)``
+            - ``Term getAbduct(Term conj)``
+            - ``Term getAbduct(Term conj, Grammar grammar)``
```

Comment:
Like above, can we use `getAbduct(Term conj, Grammar g=None)`?

Reply:
Ditto